src/CLIPreTasks/DeprecationChecker/DeprecationChecker.ts

```diff
@@ -24,9 +24,11 @@ export class DeprecationChecker {
       return
     }
 
-    const errMsg = `${chalk.bold(
-      `Your Toolbelt version (${pkgJson.version}) was deprecated`
-    )}. ${Messages.UPDATE_TOOLBELT()}`
+    const errMsg = [
+      `${chalk.bold(`Your Toolbelt version (${pkgJson.version}) was deprecated`)}.`,
+      `To deprecate, you must use the same method you used to install. As the following example(s):`,
+      ...deprecateMessageSwitch(),
```

Comment:
Is this a message for the toolbelt user, right? Why would they need to deprecate the toolbelt? Or did I misunderstand it?

Wouldn't it be better
```
To update your toolbelt, you must use the same method you used to install. (...)
```
?

Reply:
Yeah, you are right! I will fix it